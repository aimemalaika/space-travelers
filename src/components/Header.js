import React, { Component } from 'react';
import logo from '../images/planet.png';

class Header extends Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    return (
      <div className="header-container">
        <div className="logo">
          <img src={logo} alt="Logo" />
          &nbsp;&nbsp;
          Time Travellers
        </div>
        <div className="menus">
          <ul>
            <li>Rockets</li>
            <li>Mission</li>
            <li className="my-profile">My Profile</li>
          </ul>
        </div>
      </div>
    );
  }
}

export default Header;
