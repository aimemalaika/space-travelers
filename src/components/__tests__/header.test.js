import React from 'react';
import {
  BrowserRouter,
} from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../Header';

it('renders correctly', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Header />
    </BrowserRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
