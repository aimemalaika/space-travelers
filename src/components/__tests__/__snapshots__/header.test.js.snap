// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header-container"
>
  <div
    className="logo"
  >
    <img
      alt="Logo"
      src="planet.png"
    />
       Time Travelers
  </div>
  <div
    className="menus"
  >
    <ul>
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Rockets
        </a>
      </li>
      <li>
        <a
          className="inactive"
          href="/mission"
          onClick={[Function]}
        >
          Mission
        </a>
      </li>
      <li
        className="my-profile"
      >
        <a
          className="inactive"
          href="/profile"
          onClick={[Function]}
        >
          My Profile
        </a>
      </li>
    </ul>
  </div>
</div>
`;
